src/main/java/bio/terra/service/DatasetService.java

```diff
@@ -81,10 +83,14 @@ public String createDataset(DatasetRequestModel datasetRequestModel) {
      * @param id dataset id to delete
      * @returns jobId (flightId) of the job
      */
-    public String deleteDataset(String id) {
+    public String deleteDataset(UUID id, AuthenticatedUserRequest userInfo) {
         FlightMap flightMap = new FlightMap();
         flightMap.put(JobMapKeys.DESCRIPTION.getKeyName(), "Delete dataset " + id);
+        // TODO talk about conventions for naming data in the input map
+        // and in the step whether to pass data from input map to steps or let each step retrieve what they need
+        flightMap.put(JobMapKeys.REQUEST.getKeyName(), id);
```

Comment:
i wanted to have a discussion about conventions for handling data in flights. i noticed in your ingest flight you define keys that are used in your steps. i was trying to use JobMapKeys instead of just a string called "id". i agree that it is redundant so i'll change this to passing in the id to the step. i will leave the TODO in the code to remind us to have this conversation